### PR TITLE
Endpoint variables added to the .env file

### PR DESCRIPTION
**Pull request by niksfred**
React .env variables added for BE endpoint selection.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_LOCAL ='http://localhost:3030/'
```
Comment (Galabov93) [inline .env:1]: Add this file to .gitignore. It should not be apart of github

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "@material-ui/icons": "^4.4.3",
     "axios": "^0.21.1",
     "date-fns": "^2.14.0",
+    "dotenv": "^10.0.0",
```
Comment (Galabov93) [inline package.json:10]: Remove this package as it's not used.

File: src/config/API.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 import { logout } from "../services/Authentication";
 
-const baseURL =
-  process.env.NODE_ENV === "development"
-    ? "http://localhost:3030/"
-    : "http://ec2-3-127-248-27.eu-central-1.compute.amazonaws.com:3030";
+// require('dotenv').config();  Просто използвайки REACT_APP ни позволява да ползваме .env променливите. 
+
+// Use SERVER for AWS server endpoint or LOCAL for local database
+const baseURL = process.env.REACT_APP_SERVER;
```
Comment (Galabov93) [inline src/config/API.js:7]: Use the previous logic here, but with the process.env constants and set you REACT_APP_LOCAL to be equal to the SERVER one.

**Review by Galabov93 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (niksfred) [reply .env:1]: .env file added to the gitignore file.

File: package.json (hunk 2 above)
Comment (niksfred) [reply package.json:10]: package uninstalled and removed from package.json file.

File: src/config/API.js (hunk 3 above)
Comment (niksfred) [reply src/config/API.js:7]: Done.

**Review by Galabov93 — APPROVED** [review]

**Pull request merged**